.gitignore

```diff
@@ -111,4 +111,7 @@ venv.bak/
 data/*
 temp_private_key.pem
 .idea/
+packages
+aea_scripts
```

Comment:
@dishmop  we do not want to ignore packages

We also don't need aea_scripts here

Reply:
hmm I think that must be something I changed by accident thinking I was in the carpark repo. I'll revert this.